idd/Energy+.idd.in

```diff
@@ -78133,21 +78196,169 @@ ElectricLoadCenter:Distribution,
+       \note required field if Storage Charge Operation Scheme is set to ScheduledGridSupply or OnSiteGeneratorSurplusPlusScheduledGridSupply
+       \note Schedule values should be fractions from 0.0 to 1.0, inclusive.
+       \type object-list
+       \object-list ScheduleNames
+  A12, \field Storage Converter Object Name
+       \note name of ElectricLoadCenter:Storage:Converter used to convert AC to DC when charging DC storage from grid supply.
+       \note Required field when both buss type is DirectCurrentWithInverterDCStorage and Storage Charge Operation
+       \note Scheme is set to ScheduledGridSupply or OnSiteGeneratorSurplusPlusScheduledGridSupply.
+       \note A single bidirectional device will reference both an inverter object (DC to AC) and a converter object (AC to DC).
+       \type object-list
+       \object-list ConverterList
+  
+  A13, \field Storage Operation Scheme
+       \note Select method to govern how storage charge and discharge is controlled
+       \type choice
+       \key DemandLimit
+       \note FacilityDemandLimit indicates that storage discharge control will limit facility power demand drawn from the utility service 
+       \note while accounting for any on-site generation
+       \key TrackFacilityElectricDemand
+       \note TrackFacilityElectricDemand indicates that storage discharge control will follow the facility power demand 
+       \note while accounting for any on-site generation (legacy behavior)
+       \default TrackFacilityElectricDemand
+       \key TrackSchedule
+       \note TrackSchedule indicates that storage discharge control will follow the schedule in the Storage Discharge Track Schedule Name. 
+       \key TrackMeter
+       \note TrackMeter indicates that storage discharge control will follow an electric meter named in the field called Storage Discharge Track Meter Name.  
+       \key ScheduledGridExport
+       \note ScheduledGridExport indicates that storage discharge control will export power to the utility service connection following 
+       \note the power level in the field called Maximum Storage Discharge Grid Export Power multiplied by the value in the schedule 
+       \note named in the field Storage Discharge Grid Export Fraction Schedule Name. 
+  N5,  \field Storage Discharge Utility Demand Limit
+       \note Target utility service demand power for discharge control.  Storage draws are adjusted upwards for conversion losses. 
+       \note Required field for DemandLimit discharge operation scheme
+       \type real
+       \units W
+  A14, \field Storage Discharge Utility Demand Limit Fraction Schedule Name
+       \note Modifies the target utility service demand power over time.
+       \note Schedule values should be fractions from 0.0 to 1.0, inclusive.
+       \note if omitted a schedule value of 1.0 is used. 
+       \type object-list
+       \object-list ScheduleNames
+  A15, \field Storage Discharge Track Schedule Name
```

Comment:
Would you be open to changing this to a fraction schedule and adding a field for the max/nominal value?  Seems that experimenting with storage size would require a new schedule here which would be tedious.